### PR TITLE
Improve Add Covers as Comment

### PR DESCRIPTION
**Pull request by Yashs911**
Related to #5716 

I don't know why but on testing I can only see the covers changes done by me and not by anyone else.

I have tried to fix a few things which I noticed on testing.
Due to a PWA feature, any links with 404 image src will display OL Logo so adding height fixes the UI aspect

### Screenshot

image

### Stakeholders

@jimchamp

### CODE REVIEW

**Comment by Yashs911** [comment]
Currently on testing
image

File: openlibrary/plugins/upstream/covers.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def upload(self, key, i):
 
     def save(self, book, coverid, url=None):
         book.covers = [coverid] + [cover.id for cover in book.get_covers()]
-        book._save('{}/b/id/{}-S.jpg'.format(
+        book._save('{}/b/id/{}-M.jpg'.format(
```
Comment (jimchamp) [inline openlibrary/plugins/upstream/covers.py:87]: Was this change necessary?  It seems like this should be left as-is.

**Comment by jimchamp** [comment]
I noticed that dcapillae's image is present in the production [book page](https://openlibrary.org/books/OL23038708M/%E5%8F%B2%E5%A5%B4%E6%AF%94%E8%8B%B1%E6%BC%A2%E8%BE%AD%E5%85%B8_The_Charlie_Brown_Dictionary), but not in [testing](https://testing.openlibrary.org/books/OL23038708M/%E5%8F%B2%E5%A5%B4%E6%AF%94%E8%8B%B1%E6%BC%A2%E8%BE%AD%E5%85%B8_The_Charlie_Brown_Dictionary).  I'm wondering if he added the image in production.  Did you add your cover images in testing?

Testing has some caching issues, which may be causing this.  I'll investigate and let you know what I find out.

**Comment by Yashs911** [comment]
> Did you add your cover images in testing?

Yes, can you try adding some covers in testing (I usually download the same cover and reupload it)

File: openlibrary/plugins/upstream/covers.py (hunk 1 above)
Comment (Yashs911) [reply openlibrary/plugins/upstream/covers.py:87]: Using M size with fixed height provides better quality images so one can zoom in and see the title.

Comment (jimchamp) [reply openlibrary/plugins/upstream/covers.py:87]: I wouldn't be surprised if this has some type of side effect.  Can we change it back for now?  Happy to merge this once that is done.

**Comment by Yashs911** [comment]
@jimchamp Hey, I think after Mek's commit this PR is good to go

**Review by jimchamp — APPROVED** [review]
Lgtm!  Thanks @Yashs911!
